test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpErrorTest/CorrectlyParsesAtSignInDelimitedBlock.stree.txt

```diff
@@ -1,17 +1,22 @@
-Expression block - Gen<Expr> - 46 - (0:0,0)
-    MetaCode span - Gen<None> - [(] - SpanEditHandler;Accepts:None - (0:0,0) - Tokens:1
-        SyntaxKind.LeftParenthesis;[(];
-    Code span - Gen<Expr> - [Request["description"] ?? @photo.Description] - SpanEditHandler;Accepts:Any - (1:0,1) - Tokens:11
-        SyntaxKind.Identifier;[Request];
-        SyntaxKind.LeftBracket;[[];
-        SyntaxKind.StringLiteral;["description"];
-        SyntaxKind.RightBracket;[]];
-        SyntaxKind.Whitespace;[ ];
-        SyntaxKind.NullCoalesce;[??];
-        SyntaxKind.Whitespace;[ ];
-        SyntaxKind.Transition;[@];
-        SyntaxKind.Identifier;[photo];
-        SyntaxKind.Dot;[.];
-        SyntaxKind.Identifier;[Description];
-    MetaCode span - Gen<None> - [)] - SpanEditHandler;Accepts:None - (45:0,45) - Tokens:1
-        SyntaxKind.RightParenthesis;[)];
+CSharpCodeBlock - [0..46)::46 - [(Request["description"] ?? @photo.Description)]
+    CSharpExplicitExpression - [0..46)::46
+        CSharpTransition - [0..0)::0 - Gen<None> - SpanEditHandler;Accepts:None
+            Transition;[<Missing>];
+        CSharpExplicitExpressionBody - [0..46)::46
+            RazorMetaCode - [0..1)::1 - Gen<None> - SpanEditHandler;Accepts:None
```

Comment:
Just something I noticed now but a little odd that the explicit expression body has the `(` `)` in the body.

Reply:
Forget this comment: https://github.com/aspnet/Razor/pull/2702#discussion_r231245479